preprocessing/regex_sentence_segmentation.py

```diff
@@ -1,32 +1,31 @@
 import re
 
+RE_SPACES = re.compile("[ \n]{1,}")
```

Comment:
my mistake, `"[ \n]{1,}"` can be replaced by nicer `"[ \n]+"` or even `r"\s+"`

Reply:
I actually found it a little complicated. I'll fix it.